graphql2/graphqlapp/schedule.go

```diff
@@ -81,14 +81,22 @@ func (q *Query) Schedule(ctx context.Context, id string) (*schedule.Schedule, er
 	return (*App)(q).FindOneSchedule(ctx, id)
 }
 
-func (s *Schedule) Shifts(ctx context.Context, raw *schedule.Schedule, start, end time.Time) ([]oncall.Shift, error) {
+func (s *Schedule) Shifts(ctx context.Context, raw *schedule.Schedule, start, end time.Time, userIDs []string) ([]oncall.Shift, error) {
 	if end.Before(start) {
 		return nil, validation.NewFieldError("EndTime", "must be after StartTime")
 	}
 	if end.After(start.AddDate(0, 0, 50)) {
 		return nil, validation.NewFieldError("EndTime", "cannot be more than 50 days past StartTime")
 	}
-	return s.OnCallStore.HistoryBySchedule(ctx, raw.ID, start, end)
+
+	for _, userID := range userIDs {
+		err := validate.UUID("UserID", userID)
```

Comment:
Use `validate.ManyUUID` instead with a limit on the length

Reply:
What is the benefit of that?

Reply:
It exists specifically for this type of validation:
- It validates the length (i.e., a max number of UserIDs instead of unlimited)
- Consistency with all other code in the repo taking a slice of UUIDs already using it
- Returns the proper field name (i.e., `UserID[23]`) when there is an error on an individual value